tdmq/client/client.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import tiledb
+import numpy as np
+
+from tdmq.client.sensors import NonScalarSensor
+
+sensor_classes = {
+    'meteoRadar': NonScalarSensor,
+}
+
+
+class Client:
+    def __init__(self, tdmq_base_url, tiledb_ctx=None):
+        self.base_url = tdmq_base_url
+        self.tiledb_ctx = tiledb_ctx
+        self.sensor_types = None
+        self.update_sensor_types()
+
+    def update_sensor_types(self):
+        stypes = requests.get(f'{self.base_url}/sensor_types').json()
+        self.sensor_types = dict((st['name'], st) for st in stypes)
+
+    def get_sensors(self, args):
+        res = requests.get(f'{self.base_url}/sensors', params=args).json()
+        return [self.get_sensor_proxy(r['code']) for r in res]
+
+    def get_sensor_proxy(self, code):
+        res = requests.get(f'{self.base_url}/sensors/{code}').json()
```

Comment:
I think it is not necessary to execute another GET. According to get_sensors API https://github.com/tdm-project/tdm-polystore/blob/de6f622abe7c72eeb10969d5f382258bd6b3f979/tdmq/api.py#L70, it returns a list of JSON with all the needed info.